internal/terraform/node_resource_abstract_instance.go

```diff
@@ -640,12 +641,25 @@ func (n *NodeAbstractResourceInstance) refresh(ctx EvalContext, deposedKey state
 			NewState: priorVal,
 		}
 	} else {
+		// TODO: This should also allow the option created in TF-13948
+		deferralAllowed := ctx.LanguageExperimentActive(experiments.UnknownInstances)
 		resp = provider.ReadResource(providers.ReadResourceRequest{
-			TypeName:     n.Addr.Resource.Resource.Type,
-			PriorState:   priorVal,
-			Private:      state.Private,
-			ProviderMeta: metaConfigVal,
+			TypeName:        n.Addr.Resource.Resource.Type,
+			PriorState:      priorVal,
+			Private:         state.Private,
+			ProviderMeta:    metaConfigVal,
+			DeferralAllowed: deferralAllowed,
 		})
+
+		if resp.Deferred != nil {
+			deferrals := ctx.Deferrals()
+			expectedValue := cty.UnknownVal(cty.DynamicPseudoType)
+			deferrals.ReportResourceInstanceDeferred(absAddr, plans.Read, expectedValue)
+
+			return &states.ResourceInstanceObject{
```

Comment:
Can you add a TODO here for adding this into the plan when we have support for that? I think there's a jira ticket for this in the same epic.